src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -117,7 +117,7 @@ private readonly Dictionary<RestoreTargetGraph, Dictionary<string, LibraryInclud
         private const string AuditDurationOutput = "Audit.Duration.Output";
         private const string AuditDurationTotal = "Audit.Duration.Total";
 
-        private readonly bool _enableNewDependencyResolver;
+        private readonly bool _enableNewDependencyResolver = true;
```

Comment:
I'm not a fan of this change. 
readonly values with defaults set outside of the constructor can lead to bugs. 
It's best if the property is set in a single place like the constructor.

Reply:
Okay